libraries/engage/locations/selectors/index.js

```diff
@@ -98,6 +95,8 @@ export function makeGetUserLocationFulfillmentMethod(): Selector<
   );
 }
 
+export { makeGetUserLocationFulfillmentMethod };
```

Comment:
The function can be immediately exported. No need to right this statement

Reply:
Fixed.